### ISSUE TITLE
mod_authentication: move "logon bottom" into logon box

### ISSUE THREAD

**Issue by ArthurClemens** [zotonic/zotonic#861]
Now the link "I don’t have an account, please sign me up" is somewhere on the page. Should be grouped with "forgot password".

The logon box should be structured better to style it properly.

**Comment by ArthurClemens** [comment]
I have started work on having the screens (optionally) in a modal dialog (``_logon_modal.tpl``). Because this can be accessed from anywhere on the site, direct form submits to controller_logon are not possible. In this case I am using a wired postback.

Sign up in the modal still needs to be done (large form).

I have documented the template structure in ``/modules/mod_authentication/templates/__template_structure.txt``. Of three templates I cannot find where they are used::

* _authentication_services_connected.tpl
* _logon_off.tpl
* logon_confirm_form.tpl

Error messages are really long. I can shorten them, and remove the current texts from the po files. As long as this is in master it won't cause problems.
